frontend_v2/src/app/components/challenge/challenge.component.html

```diff
@@ -15,76 +15,66 @@
                                 </div>
                                 <div class="col-lg-7 col-md-6">
                                     <div class="card-content">
-                                        <p>
-                                            <span style="line-height: 1.5" class="fs-18">
-                                                <strong style="color: rgba(0, 0, 0, 0.87);"
-                                                    class="fw-semibold">
+                                            <h4 class="challenge-title fw-light">
                                                     {{challenge['title']}}
-                                                </strong>
-                                            </span>
-                                            &nbsp;
                                             <span *ngIf="isChallengeHost">
                                                 <a style="padding-right: 4px;"
                                                     class="pointer"
                                                     (click)="editChallengeTitle()">
-                                                    <i class="fa fa-pencil"
+                                                    <i class="fa fa-pencil fs-16"
                                                         aria-hidden="true"></i>
                                                 </a>
                                                 <a class="pointer"
                                                     (click)="deleteChallenge()">
-                                                    <i class="fa fa-trash red-text"
+                                                    <i class="fa fa-trash red-text fs-16"
                                                         aria-hidden="true"></i>
                                                 </a>
                                             </span>
-                                            <br>
-                                            <span style="line-height: 1.5"
+                                            </h4>
+                                            <span style="line-height: 1.5" class="fw-light fs-16"
```

Comment:
Hey @Kajol-Kumari, Let's not use inline CSS. Can you please create a class and add this.